docs/source/architecture/poet.rst

```diff
@@ -571,7 +572,7 @@ Security Considerations
       elapses. This again prevents a malicious user from running multiple
       instances of the enclave to create multiple WaitCertificates (each with a
       different nonce) resulting in different WaitCertId digests and selecting
-      the one that would result in the lowest duration for a new WaitTimer.
+      the one that would result in the lowest duration of its next WaitTimer.
```

Comment:
"`for a new` WaitTimer" sounds rightly worded. The intent is to stop the scenario where multiple instances of enclave are run to get multiple WaitTimer for the same block. Not necessarily for a new (block's) WaitTimer.